contracts/storage/RibbonThetaSTETHVaultStorage.sol

```diff
@@ -30,14 +30,20 @@ abstract contract RibbonThetaSTETHVaultStorageV3 {
     address public liquidityGauge;
 }
 
+abstract contract RibbonThetaSTETHVaultStorageV4 {
+    // Queued withdraw shares for the current round
+    uint256 public queuedWithdrawShares;
```

Comment:
I think we should consider renaming this to `currentRoundQueuedWithdrawShares` or something clearer because we use the same name in `vaultParams`

Reply:
lol true, just fixed